content/members.md

```diff
@@ -24,28 +24,23 @@ You can contact us at contact(at)fink-broker.org for more information.
 
 #### Argentina
 
-* **International Center for Advanced Studies, Physical Sciences Institute, San Martin University**: M. Makler, K. Nowogrodzki
+* **International Center for Advanced Studies, Physical Sciences Institute, San Martin University**: M. Makler
 * **UNAHUR/CONICET**: G. J. M. Luna
 
 #### Australia
-* **Astronomy Data And Computing Services**: J. Hu
-* **Australian National University**: C. Lidman, B. Schmidt, J. Soon
+* **Australian National University**: C. Lidman, B. Schmidt
 * **ICRAR-Curtin University**: A. Goodwin
 * **Monash University**: R. Hirai (also RIKEN, Japan)
-* **Swinburne University fo Technology**: A. Möller, D. Dobie, C. Gately, G. Poole, H. Roxburgh, S. Saleheen, R. Seikel, L. Steinwender, N. Van Bemmel, T. Waterson, S. Webb
-* **The University of Adelaide**: S. Einecke
+* **Swinburne University fo Technology**: N. Van Bemmel, R. Seikel, L. Steinwender,  O. V. Velazquez, T. Waterson, S. Webb
 * **The University of Melbourne**: K. Auchettl
-* **The University of New South Wales Canberra**: A. Ruiter
-* **The University of Queensland**: N. Khetan
 * **The University of Sydney**: K. Rose
 
 #### Brazil
 
-* **Centro Brasileiro de Pesquisas Físicas**: J.P. Correia de França, C.R. De Bom, B. Fraga, J.C. Rodríguez Ramírez, A. Santos, G.S.M. Teixeira
```

Comment:
Gabriel should be here (sent his form for the first time few days ago)

Reply:
added